blocklylib-core/src/main/java/com/google/blockly/model/Connection.java

```diff
@@ -410,6 +410,24 @@ private boolean checksMatch(Connection target) {
         return false;
     }
 
+
+    public static boolean checksMatch(Connection source,Connection target){
```

Comment:
Please add a JavaDoc comment on this method.

Reply:
@AnmAtAnm OK , I  pushed a commit just now.